src/views/Login.jsx

```diff
@@ -77,11 +77,11 @@ class Login extends Component {
         />
         <br/>
         <RaisedButton
-          onClick={this.onSubmit}
+          onSubmit={this.handleOnSubmit}
```

Comment:
Again, this shouldn't be a form. For a raised button, you would want to use the `onTouchTap` event (instead of `onClick`)